src/gui/tray/UserModel.cpp

```diff
@@ -646,11 +643,13 @@ Q_INVOKABLE void UserModel::openCurrentAccountTalk()
     if (_users.isEmpty())
         return;
 
-    QString url = _users[_currentUserId]->server(false) + "/apps/spreed";
-    if (!(url.contains("http://") || url.contains("https://"))) {
-        url = "https://" + _users[_currentUserId]->server(false) + "/apps/spreed";
+    auto talkApp = _users[_currentUserId]->talkApp();
```

Comment:
I'd say use `currentUser()` instead of `_users[_currentUserId]`. We're duplicating that logic everywhere inside `UserModel` otherwise.